SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityDeployHealthHelper.java

```diff
@@ -246,6 +246,12 @@ public DeployHealth getTaskHealth(SingularityDeploy deploy, boolean isDeployPend
         return DeployHealth.HEALTHY;
       }
     }
+    if (deploy.getHealthcheck().isPresent() && deploy.getHealthcheck().get().getExpectHealthcheckConfigFile()) {
```

Comment:
not sure how to do this part without the getExpectHealthcheckConfigFile. Not sure if it's even necessary in that case?